pkg/config/config.go

```diff
@@ -104,62 +104,6 @@ func NewConfig() *Config {
 	}
 }
 
-func (c *Config) SetApiAddress(address string) {
-	c.Api.ListeningAddress = address
-}
-
-// SetSparrowName sets the DNS name of the sparrow
-func (c *Config) SetSparrowName(name string) {
-	c.SparrowName = name
-}
-
-// SetLoaderType sets the loader type
-func (c *Config) SetLoaderType(loaderType string) {
-	c.Loader.Type = loaderType
-}
-
-func (c *Config) SetLoaderFilePath(loaderFilePath string) {
-	c.Loader.file.path = loaderFilePath
-}
-
-// SetLoaderInterval sets the loader interval
-// loaderInterval in seconds
-func (c *Config) SetLoaderInterval(loaderInterval int) {
-	c.Loader.Interval = time.Duration(loaderInterval) * time.Second
```

Comment:
Please be aware of those conversions!
Currently the time/duration related config parameters are set to seconds in those setter func. 
Same in SetLoaderHttpTimeout & SetLoaderHttpRetryDelay

I would suggest either  (A) switching the flags to `string` type (e.g loader.interval: 3 -> loader.interval: 3s) or (B) convert the duration to seconds in the loader itself.

For A: I am not sure if the viper unmarshaler can unmarshal to time.Duration directly.

Reply:
Sw from setter to the unmarshal is nice! But removing the setter of the config of time & duration will result in a major change in the config as well (old: 3 results in 3 seconds; now: 3 results in 3 nanoseconds).

Reply:
The parser should fix this on its own: https://github.com/caas-team/sparrow/pull/63#discussion_r1448949879